migrate.go

```diff
@@ -68,8 +68,7 @@ func migrateDB(config map[string]string) {
 			if err != nil || strings.Contains(string(result), "ERROR") {
 				if err == nil {
 					err = fmt.Errorf("\n%s", string(result))
-				}else {
-
+				} else {
```

Comment:
Why are we only breaking from this else?

Reply:
We are testing for `err == nil`, which will never happen. If this is not true then we need to talk.